feature_engine/selection/__init__.py

```diff
@@ -1,9 +1,10 @@
 """
-The module selection includes classes to select features or remove unwanted features
+The module selection includes classes to select features, remove unwanted features or shuffle feature values
 """
```

Comment:
I think the original sentence describes this module better:

"The module selection includes classes to select features or remove unwanted features"

Could we revert?

Reply:
sure